determined-ai/determined
# chore: format code

## Description
- Related to #5015 
- This is Step2 of https://github.com/determined-ai/determined/pull/5015#issuecomment-1248376734
- Just ran `npm run format`




## Test Plan
- Check if code works as expected
- Check if all test cases pass



## Commentary (optional)




## Checklist
See [Release Note](https://github.com/determined-ai/determined/blob/master/docs/release-notes/README.md) for details.





[DET-1234]: https://determinedai.atlassian.net/browse/DET-1234?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
